File: java/03-bootstrap/src/main/java/com/fResult/bootstrap/customers/services/ComponentScannedCustomerService.java
package com.fResult.bootstrap.customers.services;

import javax.sql.DataSource;
import org.springframework.stereotype.Service;
import org.springframework.transaction.support.TransactionTemplate;

@Service
public class ComponentScannedCustomerService extends TransactionTemplateCustomerService {
  public ComponentScannedCustomerService(
      DataSource dataSource, TransactionTemplate transactionTemplate) {

    super(dataSource, transactionTemplate);
  }
}

File: kotlin/03-bootstrap/src/main/kotlin/com/fResult/bootstrap/customers/services/ComponentScannedCustomerService.kt
package com.fResult.bootstrap.customers.services

import org.springframework.stereotype.Service
import org.springframework.transaction.support.TransactionTemplate
import javax.sql.DataSource

@Service
class ComponentScannedCustomerService(
  dataSource: DataSource,
  transactionTemplate: TransactionTemplate,
) : TransactionTemplateCustomerService(dataSource, transactionTemplate)
